src/tiledb/cloud/client.py

```diff
@@ -177,6 +177,7 @@ def list_public_arrays(
     page=None,
     per_page=None,
     async_req=False,
+    with_metadata=None,
```

Comment:
Should this default to `False`? We don't have any pydantic like validator so in theory a str or some other dtype would make it into `kwargs` and the api request

Reply:
@spencerseale I asked myself the same question. I'm defaulting to `None` so that we don't put a `with_metadata=False` in our request URL and so that we default to exactly (I hope!) same response as before.

`async_req` is a different case, it isn't passed to our API servers, but leads to a new branch in our low-level client code.

Reply:
I see, thank you for explaining that!